app/code/Magento/InventoryApi/Test/_files/source_items_rollback.php

```diff
@@ -30,5 +31,8 @@
  * In that case there is "if" which checks that SKU1, SKU2 and SKU3 still exists in database.
  */
 if (!empty($sourceItems)) {
-    $sourceItemsDelete->execute($sourceItems);
+    try {
+        $sourceItemsDelete->execute($sourceItems);
+    } catch (NoSuchEntityException $e) {
```

Comment:
Why do we need wrap this code in try catch? Looks like `sourceItemsDelete` does not throw `NoSuchEntityException `